### PR TITLE
Update password reset to use redis for mapping

### PR DESCRIPTION
**Pull request by yeoutzer**

### CODE REVIEW

File: routes/api/auth.js [hunk 1]
```diff
@@ -6,17 +6,20 @@ const crypto = require("crypto");
...
 const { check, validationResult } = require("express-validator");
 const db = require("../../services/db");
+const { redisClient } = require("../../services/redis");
 require("dotenv").config();
 
 router.use(bodyParser.urlencoded({ extended: true }));
 router.use(bodyParser.json());
 
 const resetIDEmailMap = new Map();
 const emailResetIDMap = new Map();
```
Comment (Tomashiwa) [inline routes/api/auth.js:16]: i think can remove now?

File: client/src/components/AccountResetPanel/AccountResetPanel.js [hunk 2]
```diff
@@ -39,11 +39,11 @@ function AccountResetPanel() {
 		axios
 			.post("/api/auth/authreset", { rid: rid }, config)
 			.then((res) => {
-				console.log(res.data.message);
+				//console.log(res.data.message);
```
[the remark this answers is not in the document]
Comment (yeoutzer) [reply client/src/components/AccountResetPanel/AccountResetPanel.js]: Fixed in https://github.com/CS3219-SE-Principles-and-Patterns/cs3219-project-ay2122-2122-s1-g37/pull/112/commits/7e3938a4663702cd8e20f56aa86d61bf40e1b378

File: routes/api/auth.js (hunk 1 above)
Comment (yeoutzer) [reply routes/api/auth.js]: https://github.com/CS3219-SE-Principles-and-Patterns/cs3219-project-ay2122-2122-s1-g37/pull/112/commits/7e3938a4663702cd8e20f56aa86d61bf40e1b378

**Review by Tomashiwa — APPROVED** [review]
LGTM

**Pull request merged**